src/cometh/rental-protocol/rental-offer-cancelled-handler.ts

```diff
@@ -0,0 +1,85 @@
+import { log } from "@graphprotocol/graph-ts";
+import { Nft, RentalOffer } from "../../../generated/schema";
+import { RentalOfferCancelled } from "../../../generated/ComethRentalProtocol/ComethRentalProtocol";
+
+/**
+ * event RentalOfferCancelled(
+ *    uint256 indexed nonce,
+ *    address indexed maker
+ * );
+ */
+export function handleRentalOfferCancelled(event: RentalOfferCancelled): void {
+  const rentalOffer = RentalOffer.load(
+    `${event.transaction.from.toHexString()}-${event.params.nonce}`
+  );
+
+  if (!rentalOffer) {
+    log.debug(
+      "[handleRentalOfferCancelled] RentalOffer {} does not exist, tx: {}, skipping...",
+      [event.params.nonce.toString(), event.transaction.hash.toHex()]
+    );
+    return;
+  }
```

Comment:
Unfortunately we not gonna track all the offers, since the rental protocol accepts offers of any kinda with different addresses and invalid token ids. So if the offer doesn’t exists in that point we must skip it